view/frontend/web/js/view/payment/method-renderer/twocc.js

```diff
@@ -42,7 +42,7 @@ define([
         defaults: {
             active: false,
             template: 'MercadoPago_PaymentMagento/payment/twocc',
-            ccForm: 'MercadoPago_PaymentMagento/payment/cc-form',
+            TwoCcForm: 'MercadoPago_PaymentMagento/payment/twocc-form',
```

Comment:
Poderia deixar o nome no padrão camelcase
 
```suggestion
            twoCcForm: 'MercadoPago_PaymentMagento/payment/twocc-form',
```